src/doc/rustc/src/platform-support/trusty.md

```diff
@@ -0,0 +1,53 @@
+# `aarch64-unknown-trusty` and `armv7-unknown-trusty`
+
+**Tier: 3**
+
+[Trusty] is a secure Operating System that provides a Trusted Execution
+Environment (TEE) for Android.
+
+## Target maintainers
+
+- Nicole LeGare (@randomPoison)
+- Stephen Crane (@rinon)
+- As a fallback trusty-dev-team@google.com can be contacted
+
+## Requirements
+
+This target is cross-compiled. It has no special requirements for the host.
+
+It fully supports alloc with the default allocator, and partially supports std.
+Notably, most I/O functionality is not supported, e.g. filesystem support and
+networking support are not present and any APIs that rely on them will panic at
+runtime.
```

Comment:
Until std support is implemented, let's be clear that it is WIP.
```suggestion
Support for the standard library is work-in-progress. It is expected that
they will supports alloc with the default allocator, and partially supports std.
```

Reply:
Good point, I've updated the wording.